src/style.css

```diff
@@ -1 +1,106 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  color: rgba(61, 78, 97, 1);
+}
+
+.search {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search_big {
+  height: 70px;
+
+  display: flex;
+  position: relative;
+  box-sizing: border-box;
+}
+
+.search_big:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search_big::after {
+  background-image: url('./images/Search.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  content: '';
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search__input_big {
+  padding-left: 62px;
+  height: 100%;
+  width: 100%;
+  line-height: 21px;
+  outline: none;
+  box-sizing: border-box;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
```

Comment:
You can also create element `.search__input` and then use modifier ...--big and ..--small. This way you can remove repeated code.